app/services/web-development.html

```diff
@@ -105,6 +105,33 @@ <h2 class="title my-4">Websites</h2>
           Media setup, Business Emails setup and Search Engine Optimization we create an all-round solution to set the
           ball rolling for your business.
         </p>
+
+        <div class="d-flex justify-content-between align-items-center flex-wrap">
+          <picture>
+            <img src="../images/services/bootstrap-logo.svg" alt="bootstrap logo" title="Bootstrap" class="img-fluid">
+          </picture>
+          <picture>
+            <img src="../images/services/sass.svg" alt="sass logo" title="Sass" class="img-fluid">
```

Comment:
Correct capitalization in `alt`

Reply:
Fixed all typo errors & all pages are validated in html validator